src/LanguageServer.Common/Help/ElementHelp.cs

```diff
@@ -12,30 +8,11 @@ public class ElementHelp
         /// <summary>
         ///     The property description.
         /// </summary>
-        [JsonProperty("description")]
-        public string Description { get; set; }
+        public string Description { get; init; }
 
         /// <summary>
         ///     Help link for the element (if any).
         /// </summary>
-        [JsonProperty("help")]
-        public string HelpLink { get; set; }
-
-        /// <summary>
-        ///     Load help property help from JSON.
-        /// </summary>
-        /// <param name="json">
-        ///     A <see cref="JsonReader"/> representing the JSON ("PropertyName": { "description": "PropertyDescription" }).
-        /// </param>
-        /// <returns>
-        ///     A sorted dictionary of help items, keyed by property name.
-        /// </returns>
-        public static SortedDictionary<string, ElementHelp> FromJson(JsonReader json)
```

Comment:
Moved parsing directly to the caller. Don't see any value in spreading the logic across multiple places